projectile.el

```diff
@@ -548,7 +548,7 @@ The cache is created both in memory and on the hard drive."
   "Add the currently visited file to the cache."
   (interactive)
   (let* ((current-project (projectile-project-root))
-         (abs-current-file (buffer-file-name (current-buffer)))
+         (abs-current-file (file-truename (buffer-file-name (current-buffer))))
```

Comment:
Could you please move these last two bindings inside the following `(when `.  There is no point for them unless that conditional is true.

Also remove the double call to `projectile-project-root` on this occasion.